creation/web_base/singularity_lib.sh

```diff
@@ -2114,67 +2149,66 @@ ERROR   Unable to access the Singularity image: $GWMS_SINGULARITY_IMAGE
-        unset LD_LIBRARY_PATH
-    fi
-    local old_path=
-    if [[ -n "$PATH" ]]; then
-        old_path=$PATH
-        info "GWMS Singularity wrapper: PATH is set to $PATH outside Singularity. This will not be propagated to inside the container instance." 1>&2
-        PATH=$_DEFAULT_PATH
-    fi
-    local old_pythonpath=
-    if [[ -n "$PYTHONPATH" ]]; then
-        old_pythonpath=$PYTHONPATH
-        info "GWMS Singularity wrapper: PYTHONPATH is set to $PYTHONPATH outside Singularity. This will not be propagated to inside the container instance." 1>&2
-        unset PYTHONPATH
-    fi
-    if [[ -n "$LD_PRELOAD" ]]; then
-        old_ld_preload=$LD_PRELOAD
-        info "GWMS Singularity wrapper: LD_PRELOAD is set to $LD_PRELOAD outside Singularity. This will not be propagated to inside the container instance." 1>&2
-        unset LD_PRELOAD
-    fi
-
-    # Add --clearenv if requested
-    [[ -n "$GLIDEIN_CONTAINER_ENV" ]] || GLIDEIN_CONTAINER_ENV=$(gwms_from_config GLIDEIN_CONTAINER_ENV "")
-    [[ -n "$GLIDEIN_CONTAINER_ENV_CLEARLIST" ]] || GLIDEIN_CONTAINER_ENV_CLEARLIST=$(gwms_from_config GLIDEIN_CONTAINER_ENV_CLEARLIST "")
-    GWMS_SINGULARITY_EXTRA_OPTS=$(env_clear "${GLIDEIN_CONTAINER_ENV}" "${GWMS_SINGULARITY_EXTRA_OPTS}")
-    env_clearlist "$GLIDEIN_CONTAINER_ENV_CLEARLIST"
+# Duplicate of env_clear
+#    # Always disabling outside LD_LIBRARY_PATH, PATH, PYTHONPATH and LD_PRELOAD to avoid problems w/ different OS
+#    # Singularity is supposed to handle this, but different versions behave differently
+#    # Restore them only if continuing after the exec of singularity failed (end of this function)
+#    local old_ld_library_path=
+#    if [[ -n "$LD_LIBRARY_PATH" ]]; then
+#        old_ld_library_path=$LD_LIBRARY_PATH
+#        info "GWMS Singularity wrapper: LD_LIBRARY_PATH is set to $LD_LIBRARY_PATH outside Singularity. This will not be propagated to inside the container instance." 1>&2
+#        unset LD_LIBRARY_PATH
+#    fi
+#    local old_path=
+#    if [[ -n "$PATH" ]]; then
+#        old_path=$PATH
+#        info "GWMS Singularity wrapper: PATH is set to $PATH outside Singularity. This will not be propagated to inside the container instance." 1>&2
+#        PATH=$_DEFAULT_PATH
+#    fi
+#    local old_pythonpath=
+#    if [[ -n "$PYTHONPATH" ]]; then
+#        old_pythonpath=$PYTHONPATH
+#        info "GWMS Singularity wrapper: PYTHONPATH is set to $PYTHONPATH outside Singularity. This will not be propagated to inside the container instance." 1>&2
+#        unset PYTHONPATH
+#    fi
+#    if [[ -n "$LD_PRELOAD" ]]; then
+#        old_ld_preload=$LD_PRELOAD
+#        info "GWMS Singularity wrapper: LD_PRELOAD is set to $LD_PRELOAD outside Singularity. This will not be propagated to inside the container instance." 1>&2
+#        unset LD_PRELOAD
+#    fi
```

Comment:
I see this block of code being commented out and it says duplicate of something. Was this intended to be commented out and not removed if it is indeed a duplicate of something? (same question with the two code blocks commented out following this block)

Reply:
For now just commented. Not ideal but OK I think. Will remove after Jeff tests it